services/app-api/src/resolvers/questionAnswers/createQuestion.test.ts

```diff
@@ -0,0 +1,303 @@
+            query: INDEX_QUESTIONS,
+            variables: {
+                input: {
+                    pkgID: submittedPkg.id,
+                },
+            },
+        })
+
+        // Expect package to have two questions
+        expect(indexQuestionsResult?.data?.indexQuestions).toEqual(
+            expect.objectContaining({
+                DMCOQuestions: expect.objectContaining({
+                    totalCount: 2,
+                    edges: expect.arrayContaining([
+                        {
+                            node: expect.objectContaining({
+                                id: expect.any(String),
+                                dateAdded: expect.any(Date),
+                                pkgID: submittedPkg.id,
+                                documents: [
+                                    {
+                                        name: 'Test Question 1',
+                                        s3URL: 'testS3Url1',
+                                    },
+                                ],
+                                addedBy: testUserCMS,
+                            }),
+                        },
+                        {
+                            node: expect.objectContaining({
+                                id: expect.any(String),
+                                dateAdded: expect.any(Date),
+                                pkgID: submittedPkg.id,
+                                documents: [
+                                    {
+                                        name: 'Test Question 2',
+                                        s3URL: 'testS3Url2',
+                                    },
+                                ],
+                                addedBy: testUserCMS,
+                            }),
+                        },
+                    ]),
+                }),
+                DMCPQuestions: expect.objectContaining({
+                    totalCount: 0,
+                    edges: [],
+                }),
+                OACTQuestions: expect.objectContaining({
+                    totalCount: 0,
+                    edges: [],
+                }),
+            })
+        )
+    })
+    it('returns an error package status is DRAFT', async () => {
+        const stateServer = await constructTestPostgresServer()
+        const cmsServer = await constructTestPostgresServer({
+            context: {
+                user: testUserCMS,
+            },
+        })
+
+        const newSubInput: CreateHealthPlanPackageInput = {
+            programIDs: [
+                '5c10fe9f-bec9-416f-a20c-718b152ad633',
+                '037af66b-81eb-4472-8b80-01edf17d12d9',
+            ],
+            riskBasedContract: false,
+            submissionType: 'CONTRACT_ONLY',
+            submissionDescription: 'A real submission',
+            contractType: 'BASE',
+        }
+
+        const draftPkg = await stateServer.executeOperation({
```

Comment:
We should have a helper test function for this, `createTestHealthPlanPackage`, any reason not to use that instead of manually executing the CREATE_HEALTH_PLAN_PACKAGE request?

Reply:
In general, I’ve mostly found that I only use the .executeOperation fucntion on the resolver i’m actively testing and try and use test helper functions for everything else.